vnext/ReactCopies/RNTester/js/examples/Image/ImageExample.js

```diff
@@ -26,7 +26,7 @@ const base64Icon =
 
 const ImageCapInsetsExample = require('./ImageCapInsetsExample');
 const IMAGE_PREFETCH_URL =
-  'http://origami.design/public/images/bird-logo.png?r=1&t=' + Date.now();
+  'https://www.facebook.com/favicon.ico?r=1&t=' + Date.now();
```

Comment:
ReactCopies needs to remain an exact copy of upstream RN, and will get overwritten during updates.

Reply:
What version was the change in? If we need to add a new version of this to our fork we can, but we need to do it a different way.

Reply:
Looks like we'll get this with 63. Didn't look to see if it all came into nightly bits.

Could we leave the assertion removed but leave tester as is? I'm going to start integrating a 63-ish release next week. Bad links should recoverably redbox, right?

Reply:
If this needs to match facebooks, repo, should we at least send a PR to facebook to declare this url as a private const in the file? 
It seems silly to repeat a URL many times ;)

Reply:
I mean that this exact file is a copy from React Native at a given version. There's an ask for them to publish source for this as a package, but right now we have tooling to ensure it stays up to date.

Reply:
I can revert the RNTester changes, but I only updated it because one of the urls they changed is now dead: https://facebook.github.io/react-native/img/favicon.png

Reply:
Can definitely understand the intent here, but forking is expensive in general, and we should avoid it where possible. Can explain more about that if interested. 

Do the wrong URLs prevent validating the change? That's a case where we might be okay making changes, but we need to do it in a slightly different way. 

But in general, my preference would be to wait until we get Facebook's, especially since it's coming in the version we're about to upgrade to.